activejob/test/support/que/inline.rb

```diff
@@ -3,7 +3,11 @@
 Que::Job.class_eval do
   class << self; alias_method :original_enqueue, :enqueue; end
   def self.enqueue(*args)
-    args.pop if args.last.is_a?(Hash)
+    if args.last.is_a?(Hash)
+      options = args.pop
+      options.delete(:run_at)
```

Comment:
Que supports arg hashes like enqueue('string_arg', my_arg: 'my_value', run_at: 1.day.from_now) where the run_at option is pulled out and used by Que to schedule the job and the remaining args are passed through to the job, so this mimics more closely how the gem actually operates. My changes to the adapter were causing tests to fail, because the previous naive solution of just popping off the final arg if it's a hash didn't work once there weren't any arguments other than the serialized job itself.

Really, I suppose we could just pass the args straight through, it may not ever foul up a test case to have an extra :run_at param passed into a test job, but it seemed safer to mimic the gem's actual behavior.